packages/servicenow_cmdb/data_stream/event/manifest.yml

```
@@ -0,0 +1,498 @@
+title: Event
+type: logs
+streams:
+  - input: cel
+    title: ServiceNow tables
+    description: Collect table logs from ServiceNow CMDB instance via API.
+    template_path: cel.yml.hbs
+    enabled: false
+    vars:
+      - name: url
+        type: url
+        title: URL
+        description: URL of ServiceNow instance. It should follow the given format, https://<instance_id>.service-now.com.
```

```suggestion
        description: URL of ServiceNow instance. It should follow the given format, https://\<instance_id\>.service-now.com.
```
(please check whether this is necessary)

---

Similar question as asked in the readme